myhpi/tests/polls/test_ranked_choice_algorithm.py

```diff
@@ -0,0 +1,107 @@
+from datetime import datetime, timedelta
+
+from django.test.utils import tag
+
+from myhpi.polls.models import (
+    PollList,
+    RankedChoiceBallot,
+    RankedChoiceBallotEntry,
+    RankedChoiceOption,
+    RankedChoicePoll,
+)
+from myhpi.tests.core.utils import MyHPIPageTestCase
+
+
+class RankedChoiceAlgorithmTests(MyHPIPageTestCase):
+    def setUp(self):
+        super().setUp()
+
+        method = getattr(self, self._testMethodName)
+        tags = getattr(method, "tags", {})
+        if "skip_setup" in tags:
+            return
+        self.setup_poll()
+
+        self.options = {
+            "alice": RankedChoiceOption.objects.create(name="Alice", poll=self.poll),
+            "bob": RankedChoiceOption.objects.create(name="Bob", poll=self.poll),
+            "charlie": RankedChoiceOption.objects.create(name="Charlie", poll=self.poll),
+            "dora": RankedChoiceOption.objects.create(name="Dora", poll=self.poll),
+        }
+
+    def setup_poll(self):
+        self.poll_list = PollList(
+            title="Polls",
+            slug="polls",
+            path="0001000200010005",
+            depth=4,
+            is_public=True,
+        )
+        self.information_menu.add_child(instance=self.poll_list)
+
+        self.poll = RankedChoicePoll(
+            title="How are you?",
+            slug="how-are-you",
+            description="This is a poll to check how you are.",
```

Comment:
Adjust text for the options to make more sense

Reply:
Fixed by eebe3ce9c8b24b48c6e0f6ccd5e9cf13e7454835